README.md

```diff
@@ -1,4 +1,67 @@
 ![CI](https://github.com/flxo/logd-logger/workflows/CI/badge.svg)
 
 logd-logger
-===========
\ No newline at end of file
+===========
+A library to simplify logging on Android.
+
+The library provides a logd implementation for the API of 
+[log](https://crates.io/crates/log).
+Only one logging implementation may exist at runtime for rusts log API.
+Logd-logger is configured once when initiating.
+Messages are sent using the macros `error!`, `warn!`, `info!`, `debug!` and `trace!`.
+
+## Android
+In case logd-logger is compiled for Android, 
+the log messages are sent to Android's logd socket.
+
+## Other OS
+For other Operating Systems the logging messages are sent to standard out.
+These are displayed in logd format.
+
+# Usage
+
+First, add this to your Cargo.toml
+
+```toml
+[dependencies]
+logd-logger = "0.1.0"
+```
+
+Next:
+In the example the logging is initialized. First a [`Builder`] is created.
+Then the log level is set. All messages with lower log level are dropped.
+Then a tag is set which is added in front of each log message.
+Next, the module name is set to be prepended.
+After the configuration, it is tried to initialize the global logger.
+
+Afterwards five different macros can be used for logging.
```

Comment:
This can be omitted since everyone who uses this crates know the Rust logger concept.